app/controllers/sellers_controller.rb

```diff
@@ -11,14 +11,17 @@ def index
       raise InvalidParameterError, query.errors.full_messages.to_sentence
     end
 
-    @sellers = Seller.order("#{query.sort_key} #{query.sort_order}")
-
-    sellers = @sellers.map do |seller|
+    Rails.cache.fetch("sellers", expires_in: 60.seconds) do
```

Comment:
There was some offline discussion about this -- but I think we can _way_ up this expiry time -- 60s solves for the forward / back case, but I think 1hr (or even 2hrs) is even better to just overall speed up most page loads.

Eventually I think we could even create a background job to keep it fresh and spare the person who gets the cache miss

Reply:
will this cause donation totals to be off?

Reply:
It will cause them to be off, or at least "lag" behind what's really happening. But for the sake of performance, Ling, Nanxi, and Brian all signed off I believe.

Reply:
increased to 1 hour